src/main/java/com/triangledeterminer/strategy/TriangleSidesStrategy.java

```diff
@@ -0,0 +1,38 @@
+package com.triangledeterminer.strategy;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import com.triangledeterminer.model.Triangle;
+import com.triangledeterminer.model.TriangleStrategy;
+import com.triangledeterminer.model.Type;
+
+public class TriangleSidesStrategy implements TriangleStrategy {
+	private Logger logger = LogManager.getLogger(TriangleSidesStrategy.class);
+
+	@Override
+	public Type determineType(Triangle triangle) {
+		float side1 = triangle.getSide1();
```

Comment:
Do we need to initialize three more float variables here?

Reply:
Not necessarily